dist/rpm/openQA.spec

```diff
@@ -583,9 +583,6 @@ fi
 %dir %{_sysconfdir}/openqa
 %dir %{_sysconfdir}/openqa/openqa.ini.d
 %dir %{_sysconfdir}/openqa/database.ini.d
-%dir %{_distconfdir}/openqa
```

Comment:
The worker and client config need a similar change.

Reply:
done